### PR TITLE
fix(dashboard): Add tooltips icons on space dashboard…

### PR DESCRIPTION
**Pull request by Bumbilo**
…board #4294

- Added tooltips for buttons on the dashboard

screenshot from 2018-12-18 19-04-38
screenshot from 2018-12-18 19-05-13
screenshot from 2018-12-18 19-06-18

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @Bumbilo for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin.

**Comment by christianvogt** [comment]
@Bumbilo I just saw that UX wanted some changes to the issue you just did.... sorry! i had no idea they were doing so they removed some of the + icons and instead added info icons. Could you look at moving the tooltips to new info icons and removing the + icons where they indicate.
I just asked them to confirm if these are the final changes so we don't have to make more changes
have a look at the new changes and let me know if you have any questions.

File: packages/fabric8-ui/src/app/dashboard-widgets/pipelines-widget/pipelines-widget.module.ts [hunk 1]
```diff
@@ -3,14 +3,22 @@ import { NgModule } from '@angular/core';
...
+    FormsModule,
+    LoadingWidgetModule,
+    RouterModule,
+    MomentModule,
+    TooltipModule.forRoot()
+  ],
   declarations: [PipelinesWidgetComponent],
   exports: [PipelinesWidgetComponent],
-  providers: [PipelinesService],
+  providers: [PipelinesService, TooltipConfig]
```
Comment (joshuawilson) [inline packages/fabric8-ui/src/app/dashboard-widgets/pipelines-widget/pipelines-widget.module.ts]: Prettier wants a comma at the end of line 18 and 22.

File: packages/fabric8-ui/src/app/dashboard-widgets/create-work-item-widget/create-work-item-widget.module.ts [hunk 2]
```diff
@@ -19,9 +20,11 @@ import { CreateWorkItemWidgetComponent } from './create-work-item-widget.compone
     PlannerListModule,
     NgArrayPipesModule,
     WorkItemDetailModule,
-    NgLetModule,
+    TooltipModule.forRoot(),
+    NgLetModule
   ],
   declarations: [CreateWorkItemWidgetComponent],
   exports: [CreateWorkItemWidgetComponent],
+  providers: [TooltipConfig]
```
Comment (joshuawilson) [inline packages/fabric8-ui/src/app/dashboard-widgets/create-work-item-widget/create-work-item-widget.module.ts]: Prettier wants a comma at the end of line 24 and 28

File: packages/fabric8-ui/src/app/dashboard-widgets/add-codebase-widget/add-codebase-widget.module.ts [hunk 3]
```diff
@@ -15,8 +16,10 @@ import { CodebaseItemModule } from './codebase-item/codebase-item.module';
     FeatureFlagModule,
     LoadingWidgetModule,
     CodebaseItemModule,
+    TooltipModule.forRoot()
   ],
   declarations: [AddCodebaseWidgetComponent],
   exports: [AddCodebaseWidgetComponent],
+  providers: [TooltipConfig]
```
Comment (joshuawilson) [inline packages/fabric8-ui/src/app/dashboard-widgets/add-codebase-widget/add-codebase-widget.module.ts]: Prettier wants a comma at the end of line 19 and 23.

**Comment by joshuawilson** [comment]
Please rebase.

**Pull request merged**